include/odp/api/spec/classification.h

```diff
@@ -671,7 +671,13 @@ typedef struct odp_cls_cos_param {
 			odp_pktin_hash_proto_t hash_proto;
 		};
 	};
-	/** Pool associated with CoS */
+	/** Pool associated with CoS
+	 *
+	 * May be set to ODP_POOL_INVALID, in which case the originating pktin
+	 * pool is used.
```

Comment:
"originating pktin pool" is not well defined when classifier in entered through IPsec (odp_ipsec_sa_param_t::dest_cos). The packet being classified may not have originated in a pktin at all.

Reply:
Could mention odp_pktio_open() and refer the pool as packet input "default pool".

Reply:
v2.